infer/src/java/jMain.ml

```diff
@@ -35,21 +35,29 @@ let do_source_file program tenv source_basename package_opt source_file =
   store_icfg source_file cfg
 
 
+let do_class tenv program cn node =
+  let class_source_file = SourceFile.from_abs_path (JFrontend.classname_path cn) in
+  L.(debug Capture Medium)
```

Comment:
How important is this piece of logging? Just want to make sure that it won't be polluting debug logs too much

Reply:
Nvm, just realised that we're doing the same thing in `do_source_file` :)